recipe/patches/0003-add-break_cycles-for-mmapped-file-in-test_wavfile.patch

```diff
@@ -0,0 +1,40 @@
+From 69071f9162d476afaefbe9117e4b12fd2893608a Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Thu, 5 Aug 2021 20:03:24 +0200
+Subject: [PATCH 3/3] add break_cycles for mmapped file in test_wavfile
+
+---
+ scipy/io/tests/test_wavfile.py | 10 +++++++++-
+ 1 file changed, 9 insertions(+), 1 deletion(-)
+
+diff --git a/scipy/io/tests/test_wavfile.py b/scipy/io/tests/test_wavfile.py
+index ddf682c15..cac853fd5 100644
+--- a/scipy/io/tests/test_wavfile.py
++++ b/scipy/io/tests/test_wavfile.py
+@@ -1,10 +1,11 @@
+ import os
++import shutil
+ import sys
+ from io import BytesIO
+ 
+ import numpy as np
+ from numpy.testing import (assert_equal, assert_, assert_array_equal,
+-                           suppress_warnings)
++                           break_cycles, suppress_warnings, IS_PYPY)
+ import pytest
+ from pytest import raises, warns
+ 
+@@ -408,3 +409,10 @@ def test_write_roundtrip(realfile, mmap, rate, channels, dt_str, tmpdir):
```

Comment:
Did you forget the pytest.parameterize decorators here?

Reply:
They're in the previous patch; this is just how `git format-patch` creates the file...